README.md

```diff
@@ -231,7 +231,7 @@ unleash.once('changed', () => {
     console.log(`Demo is enabled: ${unleash.isEnabled('Demo')}`);
 });
 
-unleash.on('count', (name, enabled) => console.log(`isEnabled(${name}`)
+unleash.on('count', (name, enabled) => console.log(`isEnabled(${name}`));
```

Comment:
Nice catch! But I think there's another closing parenthesis missing 🤓  Like this, maybe?
```suggestion
unleash.on('count', (name, enabled) => console.log(`isEnabled(${name})`));
```